app/assets/javascripts/Components/Modals/send_lti_grades_modal.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import Modal from "react-modal";
+
+class LtiGradeModal extends React.Component {
+  static defaultProps = {
+    override: false,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      deployments: this.props.lti_deployments,
```

Comment:
If you're not updating this value ever there is no need to store it in the state. Just refer to the prop directly